File: LanPlatform/platformJS/Content/ContentAccess.js
LPContent.ContentAccessType = {
    None: 0,
    User: 1,
    Group: 2
}

LPContent.ContentAccess = function ()
{
    this.Id = 0;
    this.Version = 0;
    this.Content = 0;
    this.Type = LPContent.ContentAccessType.None;
    this.User = 0;
}

File: LanPlatform/Content/ContentAccess.cs
using GabionPlatform.Database;

namespace GabionPlatform.Content
{
    public class ContentAccess : EditableDatabaseObject
    {
        public long Content { get; set; }
        public ContentAccessType Type { get; set; }
        public long User { get; set; }

        public ContentAccess()
        {
            Content = 0;
            Type = ContentAccessType.None;
            User = 0;
        }
    }

    public enum ContentAccessType
    {
        None = 0,
        User,
        Role
    }
}